src/decorator/options/ColumnCommonOptions.ts

```diff
@@ -53,6 +53,6 @@ export interface ColumnCommonOptions {
      * Specifies a value transformer that is to be used to (un)marshal
      * this column when reading or writing to the database.
      */
-    valueTransformer?: ValueTransformer<any, any>;
+    transformer?: ValueTransformer<any, any>;
```

Comment:
I think you can remove those generic parameters from `ValueTransformer`. I don't really see value in them

Reply:
@pleerock It ensures that the `from` and the `to` method are compatible, which can be handy in some cases. Take a look at the example I made in response to your comment on the `ValueTransformer`-interface itself.

Reply:
I don't think it really brings any value in practice, it just makes things complicated in this particular cases in my opinion